### ISSUE TITLE
Stop scroll to top of page after keyboard hides?

### ISSUE THREAD

**Issue by steveclarke** [ionic-team/cordova-plugin-ionic-keyboard#84]
When the keyboard hides after input, my app is automatically **scrolled to the top of the screen**. I don't see an option to prevent this.

I replaced cordova-plugin-keyboard with this plugin, and that project had an option `Keyboard.automaticScrollToTopOnHiding = false.` Wondering if there's something similar?

I'm using version 4.0.1 with non-Ionic app. This solved all of my main keyboard issues with the Cordova app (thanks!) with the exception of this little annoyance.

**Comment by steveclarke** [comment]
Upon further investigation it looks like this only happens on iOS 12.x. Previous iOS work as expected and don't scroll to top of screen after keyboard is hidden/input loses focus.

**Comment by ronanmcnulty** [comment]
@steveclarke Did you manage to fix this issue?

**Comment by steveclarke** [comment]
No it's still happening for me.

**Comment by MitchEff** [comment]
Yeah, I'm getting the same with both this plugin and the standard `cordova-plugin-keyboard` one. No fix found as of yet.

**Comment by MitchEff** [comment]
Update: ironically, installing `cordova-plugin-keyboard` and setting `Keyboard.automaticScrollToTopOnHiding` to true _kind of_ helps, in that it scrolls to the top, but then next render is back where it started.

I definitely wouldn't call it a solution though, and I have no idea why this isn't more widely reported. Eugh. Several days in, and I'm no closer to working out wtf is going on.

**Comment by MitchEff** [comment]
OH! Sorry about all the comments - I've worked this one out.

It's Ionic's WKWebview plugin - if you move back to Cordova's WKWebview instead, the annoying scrolling goes away.

**Comment by steveclarke** [comment]
That's not really an option for me. Ionic version is the only one that solves my UI rendering issue.

**Comment by MitchEff** [comment]
Why? All the our ionic plugins work okay.

**Comment by jcesarmobile** [comment]
By the comments, it looks like you are using `cordova-plugin-ionic-webview`, is that right? if so I think the bug is caused by it and not by the keyboard plugin.
Can you provide a sample app so it can be reproduced? I can't reproduce on Ionic apps, and you mentioned you weren't using Ionic.

**Comment by steveclarke** [comment]
@jcesarmobile Yes I'm using Vue/Quasar with the cordova-plugin-ionic-webview plugin. I'll do up a repro sample app for you in the next couple of days.

**Comment by MitchEff** [comment]
It's certainly caused by Ionic webview - mine fixed immediately when moving to the generic Cordova WKWebview

**Comment by timbru31** [comment]
Crossref to a potential issue on the webview repo: https://github.com/ionic-team/cordova-plugin-ionic-webview/issues/399

**Comment by Gillardo** [comment]
I am experiencing this issue as well, when will it be fixed?

**Comment by sajallimbu** [comment]
Any fix for it yet? I dont want to use default cordova wkwebview plugin cause of the local resource with file:// uri not being allowed which is fixed in ionic-wkwebview.
